lib/DDG/Spice/Movies/Cast.pm

```diff
@@ -0,0 +1,22 @@
+package DDG::Spice::Movies::Cast;
+
+use DDG::Spice;
+
+spice to => 'https://api.themoviedb.org/3/search/person?query=$1&api_key={{ENV{DDG_SPICE_THE_MOVIE_DB_KEY}}}&callback={{callback}}';
```

Comment:
Please change the API key to `{{ENV{DDG_SPICE_MOVIEDB_APIKEY}}}` -- that one already exists in production 👍